client/src/containers/overview/project-details/bar-chart/index.tsx

```diff
@@ -0,0 +1,73 @@
+interface BarChartProps {
+  total: number;
+  segments: {
+    value: number;
+    label: string;
+    colorClass: string;
+  }[];
+  orientation?: "horizontal" | "vertical";
+}
+
+const BarChart = ({
```

Comment:
We need this component to be agnostic and reusable in other pages, so, please, move it to the `components` folder.